src/styles/helpers/normalizeTheme.js

```diff
@@ -32,6 +32,33 @@ function normalizeColors(colors) {
     }, {});
 }
 
+function normalizeBackground(background) {
+    if (background) {
+        const dark = isDark(background);
+        const mainText = getContrastText(background);
+        const theme = {
+            background: {
+                main: background,
+                secondary: dark ? lighten(background, 0.2) : darken(background, 0.2),
+            },
+            divider: dark ? lighten(background, 0.12) : darken(background, 0.12),
+            text: {
+                main: mainText,
+                secondary: dark ? lighten(mainText, 0.3) : darken(mainText, 0.3),
+                disabled: dark ? lighten(mainText, 0.5) : darken(mainText, 0.5),
+            },
+            action: {
+                active: dark ? lighten(background, 0.54) : darken(background, 0.54),
+                hover: dark ? lighten(background, 0.1) : darken(background, 0.1),
+                selected: dark ? lighten(background, 0.2) : darken(background, 0.2),
+                disabled: dark ? lighten(background, 0.3) : darken(background, 0.3),
+            },
+        };
+        return theme;
+    }
+    return {};
```

Comment:
should we return default values here?? we must not have visual breaking changes

Reply:
Since the return of normalizeTheme has the default values, if normalizeBackground returns {} that values are not changed.

Reply:
sure, I understand ... but there is a semantic problem when you read the code, you are normalizing which means it needs to return  the normalized values or a fallback ... and you are returning `{}` ... we would change the name to `resolveCustomBackground` ... which reads better according to your implementation